src/main/java/com/google/devtools/build/lib/rules/repository/BindRule.java

```diff
@@ -61,6 +61,10 @@ public Metadata getMetadata() {
 }
 /*<!-- #BLAZE_RULE (NAME = bind, TYPE = OTHER, FAMILY = Workspace)[GENERIC_RULE] -->
 
+<em><p>Warning: use of <code>bind()</code> is not recommended. See "<a
+href="https://github.com/bazelbuild/bazel/issues/1952">Consider removing bind</a>" for a long
+discussion of its issues and alternatives.</p></em>
```

Comment:
Right indeed let's not complicate the message and leave that discussion in #1952